.all-contributorsrc

```diff
@@ -3214,6 +3214,14 @@
             ],
             "login": "étienne_bergeron",
             "name": "Étienne Bergeron"
+        },
+        {
+            "contributions": [
+                "code"
+            ],
+            "login": "nbeliy",
+            "name": "Nikita Beliy"
+            "profile": "https://github.com/nbeliy"
```

Comment:
Can you actually add yourself as contributor via the wiki?

https://github.com/bids-standard/bids-specification/wiki/Recent-Contributors#adding-yourself-as-a-contributor

Reply:
Done. Should I remove myself from all-contributorsrc?

Reply:
yes please :+1:

Reply:
Done